chapter_introduction/index_vn.md

```diff
@@ -1785,7 +1789,14 @@ we must account for the way its actions might
 impact the future observations of the agent.
 -->
 
-*dịch đoạn phía trên*
+Sự đơn giản của học ngoại tuyến có sự hấp dẫn riêng của nó.
+Ưu thế là ta chỉ cần quan tâm đến vấn đề nhận dạng khuôn mẫu mà không cần phân tâm tới những vấn đề khác.
+Nhược điểm lại là khả năng cấu thành bài toán khá hạn chế.
+Nếu bạn đã đọc loạt truyện ngắn Robots của Asimov hoặc là người có tham vọng, bạn có thể đang tưởng tượng ra trí tuệ nhân tạo không những biết đưa ra dự đoán mà còn có thể tương tác với thế giới.
+Chúng ta muốn nghĩ tới những tác nhân thông minh chứ không chỉ những mô hình dự đoán.
+Tức là ta phải nhắm tới việc chọn *hành động* chứ không chỉ đưa ra những *dự đoán*.
+Hơn thế nữa, không giống với dự đoán, hành động còn có ảnh hưởng đến môi trường.
+Nếu muốn tạo ra những tác nhân thông minh, chúng ta phải tính tới cả các hành động mà có thể ảnh hưởng tới những quan sát của tác nhân đó trong tương lai.
```

Comment:
```suggestion
Nếu muốn huấn luyện một tác nhân thông mình, chúng ta phải tính đến cách những hành động của nó có thể tác động đến những gì nó nhận lại trong tương lai.
```
Câu này khá khó dịch. Ý tác giả có lẽ là những gì mà tác nhân gây ra ảnh hưởng tới những gì nó nhận lại trong tương lai.

Reply:
Ko nghĩ ra từ nào dịch cho hay em refactor lại câu gốc luôn :))
```suggestion
Nếu muốn tạo ra những tác nhân thông minh, chúng ta phải tính tới cả việc các hành động sẽ có thể làm thay đổi chính tương lai của nó.
```

Reply:
"future observations" là những gì nó sẽ quan sát được trong tương lại hay chính là môi trường trong tương lai. Dịch nó hẳn là "tương lai của nó" thì cũng có ý đúng nhưng nó "hơi con người" và mang ý nghĩa khác hơn. Vì không ai nói là tương lai của thuật toán A ý chỉ những gì nó sẽ nhận được trong tương lai cả. Mà nghĩ tới triển vọng phát triển của thuật toán đó nhiều hơn.

Reply:
Mình túm váy lại, 'future observations of the agents":
- "quan sát của tác nhân đó trong tương lai": sát nghĩa nhưng không rõ nghĩa
- "những gì nó nhận lại trong tương lai": không sát nghĩa (nhận lại nó rộng nghĩa hơn so với quan sát) nhưng dễ hiểu
- "làm thay đổi chính tương lai của nó": không sát nghĩa nhưng ngắn và dễ hiểu

Hiện tại trong comment của @ngcthuong cũng thấy cũng đang dùng từ "nhận lại" nên nếu không ai có ý kiến mình nữa mình sẽ commit suggestion "nhận lại" của @tiepvupsu nhé.

Reply:
Thank @NguyenTung1981 nhưng mình thấy từ "nhận thức" không đúng lắm. Nó ám chỉ bản thân agent sẽ thay đổi, trong khi câu gốc ám chỉ môi trường sẽ thay đổi/phản ứng dựa trên hành động hiện tại của agent.